lib/wit/deps/filesystem/types.wit

```diff
@@ -0,0 +1,634 @@
+package wasi:filesystem@0.2.0;
```

Comment:
I find myself surprised that we support these ... or do they just end in ENOSYS somewhere?

Reply:
(Also the adapter does indeed return ENOSYS for all of these, so any guest wasm that tries to use preview1 fs stuff will fail)